File: common/src/main/java/hirezapi/enums/Language.java
package hirezapi.enums;

import lombok.Getter;

/**
 * Language.
 * @author <a href="damian@stachuofficial.pl">Damian Staszewski</a>
 * @since 1.8
 */
@Getter
public enum Language {
  English(1),
  German(2),
  French(3),
  Spanish(7),
  Latin_Spanish(9),
  Portuguese(10),
  Russian(11),
  Polish(12),
  Turkish(13);

  private final int id;
  private final String name;

  Language(final int id) {
    this.name = name().replace("_", " ");
    this.id = id;
  }
}

File: api/src/main/kotlin/hirez/enums/Language.kt
package hirez.enums

/**
 * Language.
 * @author [Damian Staszewski](damian@stachuofficial.tv)
 * @since 1.8
 */
enum class Language(val id: Int) {
	English(1),
	German(2),
	French(3),
	Spanish(7),
	Latin_Spanish(9),
	Portuguese(10),
	Russian(11),
	Polish(12),
	Turkish(13);
	
	internal val value = name.replace("_", " ")
	
	override fun toString() = value
}
